community/teams/moderation.tt

```diff
@@ -1,36 +1,41 @@
-[% WRAPPER layout.tt title="Discourse Team" handlesLayout=1 %]
+[% WRAPPER layout.tt title="Moderation Team" handlesLayout=1 %]
 
 <div class="page-title">
   <div>
     <a href="[% root%]community/index.html#governance-teams">Community</a>
     <span>→</span>
   </div>
-  <h1>Discourse Team</h1>
+  <h1>Moderation Team</h1>
 </div>
 
 <section class="governance-description">
-  This team is responsible for configuring and moderating the
-  <a href="https://discourse.nixos.org/">Discourse forum</a>.
+  This team is responsible for moderating participation on
+official community platforms in accord with <a
```

Comment:
Could `official community platforms` be a link so that the area under this team works is clearly defined?
Maybe pointing to https://nixos.org/community/index.html#communication (and adding an `id="communication"` to the `Where to talk?` section.

Also is the list of the community platforms complete?

Reply:
I resolved this. I used id="platforms". I hope that's okay.

Reply:
:+1: for the `platforms` id. You just need to also add it to the `community.tt` to the `Where to talk?` section.

Reply:
This should already be done in community/index.tt